onnxruntime/test/fuzzing/include/OnnxPrediction.h

```diff
@@ -46,17 +46,20 @@ class OnnxPrediction
   //
   OnnxPrediction(onnx::ModelProto& onnx_model);
 
-  // Deletes the prediction object 
+  // The following constructor is meant for initializing using flatbuffer model.
+  // Memory buffer pointing to the model
+  //
+  OnnxPrediction(const std::vector<char>& modelData);
+
+  // Deletes the prediction object
   //
   ~OnnxPrediction();
```

Comment:
nit: unnecessary comment. anybody working on C++ should know what a destructor does.